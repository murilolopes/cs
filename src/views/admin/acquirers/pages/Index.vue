<template>
  <b-card>
    <div class="mb-2">
      <b-row class="d-flex flex-row justify-content-between align-items-center p-0">
        <b-col sm="12" md="8" lg="8">
          <h4 v-if="$route.meta.indexObject?.datatableTitle">{{ $route.meta.indexObject.datatableTitle }}</h4>
          <slot v-else name="table-caption" />
        </b-col>
        <b-col class="d-flex flex-row align-items-center justify-content-end px-1" sm="12" md="4" lg="4">
          <slot name="table-header-actions"></slot>

          <div name="table-header-filter" v-if="canShowFiltersButton" class="mr-2 ml-2">
            <feather-icon icon="FilterIcon" size="24" @click="showFiltersButton = true" class="cursor-pointer" />
          </div>
          <label class="mb-0 mr-1">Mostrar</label>
          <v-select v-model="perPage" :options="perPageOptions" :clearable="false"
            class="per-page-selector d-flex flex-column mr-1" />
          <label class="mb-0">registros</label>
        </b-col>
      </b-row>
    </div>

    <acquirer-card v-for="(acquirer, index) in acquirers" :key="index" :acquirer-data="acquirer" editable />

    <div>
      <b-row class="mt-2">
        <b-col cols="12" sm="9" class="d-flex align-items-center justify-content-center justify-content-sm-start">
          <span class="text-muted"> {{ showing }} </span>
        </b-col>
        <b-col cols="12" sm="3" class="d-flex align-items-center justify-content-center justify-content-sm-end">
          <b-pagination v-model="currentPage" :total-rows="totalRows" :per-page="perPage" first-number last-number
            prev-class="prev-item" next-class="next-item" class="mb-0" aria-controls="my-table" :disabled="!activePagination">
            <template #prev-text>
              <feather-icon icon="ChevronLeftIcon" size="18" />
            </template>
            <template #next-text>
              <feather-icon icon="ChevronRightIcon" size="18" />
            </template>
          </b-pagination>
        </b-col>
      </b-row>
    </div>

    <datatable-filters-handler-sidebar :is-task-handler-sidebar-active="showFiltersButton" @closeSideBar="showFiltersButton = false" @updateFilterValues="updateFilterValues" />
  </b-card>
</template>

<script>
import {
  BButton,
  BBadge,
  BCard,
  BRow,
  BCol,
  VBTooltip,
  BPagination,
} from "bootstrap-vue";
import vSelect from "vue-select";
import DatatableFiltersHandlerSidebar from '@/views/common/crud/components/DatatableFiltersHandlerSidebar.vue';

import AcquirerCard from '@views/admin/acquirers/components/AcquirerCard.vue'

export default {
  name: 'OnboardingFollowUpIndex',
  components: {
    BButton,
    AcquirerCard,
    BRow,
    BCol,
    BCard,
    vSelect,
    BPagination,
    DatatableFiltersHandlerSidebar,
  },
  directives: {
    "b-tooltip": VBTooltip,
  },
  data() {
    return {
      activePagination: false,
      acquirers: [],
      datatableFilterValues: {},
      skipCurrentPage: true,
      showFiltersButton: false,
      perPage: 5,
      perPageOptions: [5, 10, 20, 50],
      totalRows: 1,
      currentPage: 1,
      sortBy: 'created_at',
    };
  },
  mounted() {
    this.getData();
  },
  methods: {
    async getData() {
      this.activePagination = false
      let params = { ...this.params }

      params.sort = `${this.sortBy}|${this.sortDesc ? "desc" : "asc"}`

      const { data } = await this.$store.dispatch('credenciadora/fetchAll', params)
      this.acquirers = data.data
      this.perPage = data.per_page || 5
      this.totalRows = data.total
      if (!this.skipCurrentPage) this.currentPage = data.current_page
      this.skipCurrentPage = true
      this.activePagination = true
    },
    showFiltersComponent() {
      this.$emit("show-filters-component");
    },
    refresh() {
      this.getData();
    },
    updateFilterValues(values) {
      this.datatableFilterValues = values;
    },
  },
  computed: {
    showing() {
      let start =
        this.currentPage === 1 ? 1 : (this.currentPage - 1) * this.perPage + 1;
      let end = start === 1 ? this.perPage : start + this.perPage - 1;
      if (end > this.totalRows) end = this.totalRows;
      if (this.totalRows === 0) start = 0

      return `Mostrando de ${start} a ${end} de ${this.totalRows} registros`;
    },
    params() {
      let params = {
        per_page: this.perPage,
        page: this.currentPage,
      };

      if (Object.keys(this.datatableFilterValues).length) params = { ...params, ...this.datatableFilterValues };

      return params;
    },
    canShowFiltersButton() {
      return (
        this.$route.meta.indexObject?.filters &&
        this.$route.meta.indexObject?.filters.length > 0
      );
    },
  },
  watch: {
    $route() {
      this.getData();
    },
    perPage: {
      handler(newValue, oldValue) {
        if (newValue == oldValue) {
          this.currentPage = 1;
          this.skipCurrentPage = false;
        }
      },
      immediate: true,
    },
    params(newValue, oldValue) {
      if (newValue != oldValue) this.getData()
    },
  },
}

</script>

<style lang="scss">
.per-page-selector { min-width: 80px; }

.div-divider > div:not(:last-child) {
  position: relative;
}

.div-divider > div:not(:last-child)::after {
  content: "";
  position: absolute;
  right: 0;
  top: 0;
  bottom: 0;
  width: 1px;
  background: #DAE1E8;
}

.div-divider > div {
  margin: 1rem 0 1rem 1rem;
  padding-right: 1rem;
}

.underline {
  text-decoration: underline;
  text-decoration-color: #003B65;
}

.line-styles {
  border: 1px solid #DAE1E8;
  border-radius: 6px;
}
</style>
