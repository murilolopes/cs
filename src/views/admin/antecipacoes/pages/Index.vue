<template>
  <div>
    <b-row>
      <b-col>
        <datatable ref="antecipacoesDatatable">
          <template #table-caption>
            <div class="d-flex justify-content-start">
              <h4>Acompanhe abaixo todas as suas operações realizadas</h4>
            </div>
          </template>
        </datatable>
      </b-col>
    </b-row>
  </div>
</template>

<script>
import { BCard, BCol, BRow } from 'bootstrap-vue';
import Datatable from '@/views/common/crud/components/Datatable';

export default {
  name: 'Antecipacoes',
  components: {
    BCard,
    BRow,
    BCol,
    Datatable,
  },
};
</script>

<style scoped></style>
