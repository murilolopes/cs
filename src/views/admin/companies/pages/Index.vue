<template>
  <div>
    <cc-datatable ref="client-datatable" :filterValues="[]">
      <template #table-caption>
        <h4>Gerencie as empresas da plataforma</h4>
      </template>
    </cc-datatable>

    <b-modal id="import-list" centered header-bg-variant="white" hide-footer>
      <div class="d-flex flex-column justify-content-center align-items-center">
        <h3 class="mt-2 text-primary">Importar lista de clientes</h3>
        <div class="d-flex flex-column align-items-center w-75">
          <cc-dropzone standalone ref="emissao" v-model.sync="file" subtitle="Envie seu arquivo no formato XLSX." acceptFormats=".xlsx" />
        </div>
        <p><a
            href='https://creditcorp-public-assets.s3.sa-east-1.amazonaws.com/credito-frete/Template+importa%C3%A7%C3%A3o+de+empresas.xlsx'>Clique
          aqui</a> para baixar o template.</p>
        <div class="mt-3 mb-3">
          <b-button class="mr-2" variant="outline-primary" @click="$bvModal.hide('import-list')">Cancelar</b-button>
          <b-button :disabled="!hasFile" variant="primary" @click="importClients">Importar</b-button>
        </div>
      </div>
    </b-modal>
  </div>
</template>

<script>
import {BButton, BCard, BCol, BModal, BRow, BTable} from 'bootstrap-vue'
import CcDatatable from '@/views/common/crud/components/Datatable.vue'
import CcDropzone from '@/@core/components/cc-dropzone/CcDropzone.vue';

export default {
  components: {
    BCard,
    BRow,
    BCol,
    BTable,
    CcDatatable,
    BModal,
    BButton,
    CcDropzone,
  },
  data() {
    return {
      file: {}
    }
  },
  methods: {
    showImportList() {
      this.$bvModal.show('import-list')
    },
    async importClients() {
      try {
        this.$swal.showLoading()
        await this.$store.dispatch('dashboard/importClients', this.file)
        this.$refs['client-datatable'].refresh()
        this.$swal.fire({
          title: 'Sucesso!',
          text: 'Lista importada com sucesso!',
          icon: 'success',
          confirmButtonText: 'Ok'
        })
        this.$bvModal.hide('import-list')
      } catch (error) {
        this.$swal.fire({
          title: 'Erro!',
          text: 'Ocorreu um erro ao importar a lista!',
          icon: 'error',
          confirmButtonText: 'Ok'
        })
      }
    },
    showModal() {
      this.$bvModal.show('import-list')
    },
  },
  computed: {
    hasFile() {
      return this.file instanceof File
    }
  }
}
</script>

<style lang="scss">

#my-table th {
  width: 10px;
  div {
    padding: 0;
    float: left;
  }
}

#my-table th[aria-colindex="4"] {
  div {
    float: none !important;
  }
}

#my-table td {
  span {
    float: left;
  }
}

#my-table td[aria-colindex="4"] {
  span {
    float: none !important;
  }
}

#my-table td[aria-colindex="5"] {
  button {
    float: right;
  }
}

#my-table th[aria-colindex="5"] {
  width: auto !important;
}
</style>
