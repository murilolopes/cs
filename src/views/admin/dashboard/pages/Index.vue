<template>
  <div>
    dashboard

    <dashboard-footer></dashboard-footer>
  </div>
</template>

<script>
import DashboardFooter from '@/views/cedente/dashboard/components/Footer'

export default {
  components: {
    DashboardFooter
  }
}
</script>
