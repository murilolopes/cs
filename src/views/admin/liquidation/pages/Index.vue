<template>
  <div>
    <datatable ref="liquidationDatatable" />

    <b-modal
      id="reportPaymentModal"
      centered
      hide-footer
      header-bg-variant="white"
    >
      <div class="d-flex flex-column justify-content-center align-items-center">
        <h1 class="mb-1">Informar pagamento</h1>
        <p class="h5 text-center mb-3">
          Esta ação informará ao sistema que a operação selecionada foi paga.
          Você tem certeza que deseja continuar?
        </p>

        <div class="d-flex flex-row justify-content-between mt-1">
          <b-button
            variant="outline-primary"
            class="mb-1 mr-1"
            @click="$bvModal.hide('reportPaymentModal')"
          >
            Cancelar
          </b-button>
          <b-button variant="primary" class="mb-1" @click="reportPayment">
            Informar pagamento
          </b-button>
        </div>
      </div>
    </b-modal>

    <b-modal
      id="bankAccountModal"
      centered
      hide-footer
      header-bg-variant="white"
    >
      <div class="d-flex flex-column p-2">
        <h1 class="mb-3">Conta bancária</h1>

        <div class="d-flex flex-column justify-content-start">
          <div class="d-flex justify-content-start">
            <div class="d-flex flex-column mr-1">
              <span class="h5">Agência</span>
              <span>{{ bankAccount.agencia }}</span>
            </div>
            <div class="d-flex flex-column mr-1">
              <span class="h5">Banco</span>
              <span>{{ bankAccount.banco?.nome }}</span>
            </div>
            <div class="d-flex flex-column">
              <span class="h5">Conta bancária</span>
              <span>{{ bankAccount.conta }}</span>
            </div>
          </div>

          <hr />

          <div class="d-flex justify-content-start">
            <div class="d-flex flex-column mr-1">
              <span class="h5">Razão social</span>
              <span>{{ solicitacao.cedente?.razao_social }}</span>
            </div>
            <div class="d-flex flex-column mr-1">
              <span>CNPJ</span>
              <span>{{ solicitacao.cedente?.cnpj }}</span>
            </div>
          </div>
        </div>

        <div class="d-flex flex-row justify-content-between mt-1">
          <b-button
            variant="primary"
            class="mt-3"
            @click="$bvModal.hide('bankAccountModal')"
          >
            Fechar
          </b-button>
        </div>
      </div>
    </b-modal>
    <operation-detail-modal />
  </div>
</template>

<script>
import ToastificationContent from "@/views/admin/liquidation/components/ToastificationContent.vue";
import Datatable from "@/views/common/crud/components/Datatable.vue";
import EventBus from "@/services/event_bus";
import { BButton } from "bootstrap-vue";
import { downloadFromResponse } from "@/@core/comp-functions/forms/cc-dropzone";
import OperationDetailModal from "../../liquidation/components/modals/OperationDetailModal.vue";

export default {
  name: "AdminLiquidationIndexPage",
  components: {
    Datatable,
    BButton,
    OperationDetailModal,
  },
  data() {
    return {
      selectedIds: [],
      bankAccount: {},
      solicitacao: {},
    };
  },
  mounted() {
    EventBus.$on("informar-pagamento", this.showReportPaymentModal);
    EventBus.$on("view-field-click", this.showBankAccountModal);
    EventBus.$on("baixar-CNAB", this.baixarCnab);
  },
  methods: {
    async baixarCnab(ids) {
      try {
        this.selectedIds = ids.map((id) => id.id);
        const response = await this.$store.dispatch(
          "admin/baixarCnab",
          this.selectedIds
        );

        downloadFromResponse(response.data, {
          mimeType: "text/plain",
          fileName: "cnab.txt",
        });
      } catch (error) {
        console.log(error);
      }
    },
    async reportPayment() {
      try {
        this.$swal.showLoading();
        await this.$store.dispatch("admin/reportPayment", this.selectedIds);
        this.$bvModal.hide("reportPaymentModal");
        this.$swal.close();
        this.$refs.liquidationDatatable.refresh();
        this.$toast({
          component: ToastificationContent,
          position: "top-right",
          props: {
            title: "Pagamento informado",
            icon: "CheckCircleIcon",
            avatarVariant: "success",
            TitleVariant: "black",
            text: "Você acaba de informar o pagamento das operações selecionadas.",
          },
        });
      } catch (error) {
        this.$swal({
          icon: "error",
          title: "Oops...",
          text: "Ocorreu um erro ao informar o pagamento",
        });
      }
    },
    showReportPaymentModal(ids) {
      this.selectedIds = ids.map((id) => id.id);
      this.$bvModal.show("reportPaymentModal");
    },
    async showBankAccountModal(item) {
      try {
        this.solicitacao = item;
        const { data } = await this.$store.dispatch(
          "admin/bankAccount",
          item.id
        );
        this.bankAccount = data;
        this.$bvModal.show("bankAccountModal");
      } catch (error) {
        console.log(error);
      }
    },
  },
};
</script>
