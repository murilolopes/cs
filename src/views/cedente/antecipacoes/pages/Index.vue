<template>
  <cc-datatable>
    <template #table-caption>
      <h4>Acompanhe abaixo todas as suas operações realizadas</h4>
    </template>
    <template #table-header-actions>
      <b-button variant="primary" @click="toCredit()">Nova Antecipação</b-button>
    </template>
  </cc-datatable>
</template>

<script>
import { BButton, BCard, BCol, BModal, BRow, BTable, BBadge } from 'bootstrap-vue';
import CcDatatable from '@/views/common/crud/components/Datatable';

export default {
  name: 'Concessoes',
  components: {
    BCard,
    BRow,
    BCol,
    BTable,
    CcDatatable,
    BModal,
    BButton,
    BBadge,
  },
  data() {
    return {};
  },
  methods: {
    toCredit() {
      this.$router.push({ name: 'dashboard' });
    },
  },
};
</script>

<style scoped></style>
