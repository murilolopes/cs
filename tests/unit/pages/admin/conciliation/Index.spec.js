import Vue from 'vue';
import Vuex from 'vuex';
import BootstrapVue, { BButton } from 'bootstrap-vue';
import { mount, shallowMount, createLocalVue } from '@vue/test-utils';
import { FeatherIcon } from 'vue-feather-icons';
import Index from '@/views/admin/conciliation/pages/Index.vue';

Vue.use(BootstrapVue);
Vue.component('feather-icon', FeatherIcon);

const localVue = createLocalVue();
localVue.use(Vuex);

describe('Index.vue', () => {
  let actions = {
    fetchCreditCardBrands: jest.fn().mockResolvedValue({ data: [] }),
    fetchAssignors: jest.fn().mockResolvedValue({ data: [] }),
    fetchAcquirersWithoutPaginate: jest.fn().mockResolvedValue({ data: [] }),
    fetchUrs: jest.fn().mockResolvedValue({
      per_page: 10,
      total: 0,
      current_page: 1,
      data: {
        data: [],
      },
    }),
  };

  const store = new Vuex.Store({
    modules: {
      admin: {
        namespaced: true,
        state: {},
        actions,
      },
    },
  });

  it('Should render correctly', () => {
    const wrapper = shallowMount(Index, { localVue, store });
    expect(wrapper.element).toMatchSnapshot();
  });

  it('Should show conciliate button disabled by default', async () => {
    const wrapper = shallowMount(Index, {
      localVue,
      store,
      propsData: {
        items: [],
        perPage: {
          value: 10,
          label: '10',
        },
        selectedRows: [],
        filters: {
          assignor: 'a',
          acquirer: 'a',
          flag: 'a',
          status: 'a',
        },
      },
    });

    await wrapper.vm.$nextTick();
    const conciliateButton = wrapper.find('#conciliateButton');

    expect(conciliateButton.attributes('disabled')).toBe('true');
  });
});
