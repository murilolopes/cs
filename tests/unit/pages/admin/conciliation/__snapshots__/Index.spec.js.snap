// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index.vue Should render correctly 1`] = `
<div>
  <div
    class="file-manager-application"
  >
    <div
      class="content-overlay"
    />
     
    <div
      class="header-navbar-shadow"
    />
     
    <div
      class="content-area-wrapper container-xxl p-0 overflow-visible"
    >
      <div
        class="sidebar-left"
      >
        <div
          class="sidebar"
        >
          <div
            class="sidebar-file-manager"
          >
            <div
              class="sidebar-inner"
            >
              <div
                class="sidebar-list"
              >
                <div
                  class="list-group p-2"
                >
                  <h3
                    class="mt-1"
                  >
                    Filtrar por
                  </h3>
                   
                  <div
                    class="mt-2"
                  >
                    <v-select-stub
                      autocomplete="off"
                      autoscroll="true"
                      calculateposition="[Function]"
                      clearable="true"
                      clearsearchonblur="[Function]"
                      clearsearchonselect="true"
                      closeonselect="true"
                      components="[object Object]"
                      createoption="[Function]"
                      dir="auto"
                      dropdownshouldopen="[Function]"
                      filter="[Function]"
                      filterable="true"
                      filterby="[Function]"
                      getoptionkey="[Function]"
                      getoptionlabel="[Function]"
                      label="label"
                      mapkeydown="[Function]"
                      ontab="[Function]"
                      options=""
                      placeholder="Cedente/grupo econÃ´mico"
                      reduce="[Function]"
                      searchable="true"
                      searchinputqueryselector="[type=search]"
                      selectable="[Function]"
                      selectonkeycodes="13"
                      transition="vs__fade"
                      uid="1"
                      value=""
                    />
                  </div>
                   
                  <div
                    class="mt-2"
                  >
                    <flat-pickr-stub
                      class="form-control"
                      config="[object Object]"
                      events="onChange,onClose,onDestroy,onMonthChange,onOpen,onYearChange"
                      placeholder="Vencimento"
                      value=""
                    />
                  </div>
                   
                  <div
                    class="mt-2"
                  >
                    <v-select-stub
                      autocomplete="off"
                      autoscroll="true"
                      calculateposition="[Function]"
                      clearable="true"
                      clearsearchonblur="[Function]"
                      clearsearchonselect="true"
                      closeonselect="true"
                      components="[object Object]"
                      createoption="[Function]"
                      dir="auto"
                      dropdownshouldopen="[Function]"
                      filter="[Function]"
                      filterable="true"
                      filterby="[Function]"
                      getoptionkey="[Function]"
                      getoptionlabel="[Function]"
                      label="label"
                      mapkeydown="[Function]"
                      ontab="[Function]"
                      options=""
                      placeholder="Credenciadora"
                      reduce="[Function]"
                      searchable="true"
                      searchinputqueryselector="[type=search]"
                      selectable="[Function]"
                      selectonkeycodes="13"
                      transition="vs__fade"
                      uid="2"
                      value=""
                    />
                  </div>
                   
                  <div
                    class="mt-2"
                  >
                    <v-select-stub
                      autocomplete="off"
                      autoscroll="true"
                      calculateposition="[Function]"
                      clearable="true"
                      clearsearchonblur="[Function]"
                      clearsearchonselect="true"
                      closeonselect="true"
                      components="[object Object]"
                      createoption="[Function]"
                      dir="auto"
                      dropdownshouldopen="[Function]"
                      filter="[Function]"
                      filterable="true"
                      filterby="[Function]"
                      getoptionkey="[Function]"
                      getoptionlabel="[Function]"
                      label="label"
                      mapkeydown="[Function]"
                      ontab="[Function]"
                      options=""
                      placeholder="Bandeira"
                      reduce="[Function]"
                      searchable="true"
                      searchinputqueryselector="[type=search]"
                      selectable="[Function]"
                      selectonkeycodes="13"
                      transition="vs__fade"
                      uid="3"
                      value=""
                    />
                  </div>
                   
                  <div
                    class="mt-2"
                  >
                    <v-select-stub
                      autocomplete="off"
                      autoscroll="true"
                      calculateposition="[Function]"
                      clearable="true"
                      clearsearchonblur="[Function]"
                      clearsearchonselect="true"
                      closeonselect="true"
                      components="[object Object]"
                      createoption="[Function]"
                      dir="auto"
                      dropdownshouldopen="[Function]"
                      filter="[Function]"
                      filterable="true"
                      filterby="[Function]"
                      getoptionkey="[Function]"
                      getoptionlabel="[Function]"
                      label="label"
                      mapkeydown="[Function]"
                      multiple="true"
                      ontab="[Function]"
                      options="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
                      placeholder="Status"
                      reduce="[Function]"
                      searchable="true"
                      searchinputqueryselector="[type=search]"
                      selectable="[Function]"
                      selectonkeycodes="13"
                      transition="vs__fade"
                      uid="4"
                      value=""
                    />
                  </div>
                   
                  <div
                    class="mt-2 d-flex justify-content-center"
                  >
                    <b-button-stub
                      class="d-flex align-items-center"
                      tag="button"
                      target="_self"
                      type="button"
                      variant="flat-primary"
                    >
                       Limpar filtros 
                      <feather-icon-stub
                        class="ml-2"
                        icon="RotateCcwIcon"
                        size="22"
                      />
                    </b-button-stub>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <div
        class="content-right"
      >
        <div
          class="content-wrapper container-xxl p-0"
        >
          <div
            class="body-content-overlay"
          />
           
          <div
            class="content-body"
          >
            <div
              class="file-manager-main-content"
            >
              <div
                class="file-manager-content-body h-100"
              >
                <div
                  class="d-flex justify-content-between align-items-center"
                >
                  <div
                    class="justify-content-between align-items-center"
                  >
                    <div
                      class="d-flex"
                    >
                      <div>
                        <b-button-stub
                          class="mr-2"
                          disabled="true"
                          id="conciliateButton"
                          tag="button"
                          target="_self"
                          type="button"
                          variant="primary"
                        >
                          
                          Dar baixa em selecionados
                          
                          <img
                            class="img-icon"
                            height="18"
                            src="@/assets/images/receipt_off_icon.svg"
                            width="18"
                          />
                        </b-button-stub>
                      </div>
                       
                      <feather-icon-stub
                        class="arrow"
                        icon="ArrowRightIcon"
                        size="24"
                      />
                       
                      <div
                        class="ml-2"
                      >
                        <span>
                           URs selecionadas 
                        </span>
                         
                        <h3>
                          R$ 0,00
                        </h3>
                      </div>
                    </div>
                     
                    <div
                      class="mt-2"
                    >
                      <b-form-checkbox-stub
                        class="mr-3"
                        id="selectAllCheckBox"
                        value="true"
                      >
                        
                        Selecionar todos
                      
                      </b-form-checkbox-stub>
                    </div>
                  </div>
                   
                  <div>
                    <b-button-stub
                      tag="button"
                      target="_self"
                      type="button"
                      variant="outline-primary"
                    >
                      
                      ConciliaÃ§Ã£o por upload
                      
                      <img
                        class="img-icon"
                        height="17"
                        src="@/assets/images/export_icon.svg"
                        width="16"
                      />
                    </b-button-stub>
                     
                    <div
                      class="d-flex align-items-center mt-2 justify-content-end"
                    >
                      <label
                        class="mb-0"
                      >
                        Mostrar
                      </label>
                       
                      <v-select-stub
                        autocomplete="off"
                        autoscroll="true"
                        calculateposition="[Function]"
                        class="per-page-selector d-inline-block mx-50"
                        clearsearchonblur="[Function]"
                        clearsearchonselect="true"
                        closeonselect="true"
                        components="[object Object]"
                        createoption="[Function]"
                        dir="auto"
                        dropdownshouldopen="[Function]"
                        filter="[Function]"
                        filterable="true"
                        filterby="[Function]"
                        getoptionkey="[Function]"
                        getoptionlabel="[Function]"
                        label="label"
                        mapkeydown="[Function]"
                        ontab="[Function]"
                        options="[object Object],[object Object],[object Object]"
                        placeholder=""
                        reduce="[Function]"
                        searchable="true"
                        searchinputqueryselector="[type=search]"
                        selectable="[Function]"
                        selectonkeycodes="13"
                        transition="vs__fade"
                        uid="5"
                        value="10"
                      />
                       
                      <label>
                        registros
                      </label>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="mt-1"
                >
                  <b-table-stub
                    class="mb-0"
                    currentpage="1"
                    emptyfilteredtext="There are no records matching your request"
                    emptytext="There are no records to show"
                    fields="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
                    filterdebounce="0"
                    filterfunction="[Function]"
                    filterignoredfields=""
                    filterincludedfields=""
                    hover="true"
                    id="conciliation-table"
                    items=""
                    labelsortasc="Click to sort Ascending"
                    labelsortclear="Click to clear sorting"
                    labelsortdesc="Click to sort Descending"
                    perpage="10"
                    responsive="true"
                    selectable="true"
                    selectedvariant="active"
                    selectmode="multi"
                    sortcompare="[Function]"
                    sortcompareoptions="[object Object]"
                    sortdirection="asc"
                    value=""
                  />
                   
                  <div
                    class="cc-datatable-header-bg px-2 py-1"
                  >
                    <div
                      class="d-flex align-items-center justify-content-end"
                    >
                      <span
                        class="h4 mr-2 mb-0"
                      >
                         Total em URs disponÃ­veis 
                      </span>
                       
                      <h2
                        class="mb-0"
                      >
                         R$ 0,00 
                      </h2>
                    </div>
                  </div>
                </div>
                 
                <div
                  class="d-flex align-items-center"
                >
                  <div
                    class="d-flex align-items-center mt-1"
                  >
                    <b-button-stub
                      class="mr-2"
                      tag="button"
                      target="_self"
                      type="button"
                      variant="flat-primary"
                    >
                      
                      Exportar para Excel
                      
                      <img
                        class="img-icon"
                        height="17"
                        src="@/assets/images/export_icon.svg"
                        width="16"
                      />
                    </b-button-stub>
                     
                    <span
                      class="mb-0 h5 mr-2"
                      style="display: none;"
                    >
                       Mostrando de 1 a 10 de 0 registros 
                    </span>
                  </div>
                   
                  <div
                    class="align-items-center mt-1 left-auto"
                  >
                    <b-pagination-stub
                      align="left"
                      arialabel="Pagination"
                      class="mb-0"
                      disabled="true"
                      ellipsistext="â¦"
                      firstnumber="true"
                      firsttext="Â«"
                      labelfirstpage="Go to first page"
                      labellastpage="Go to last page"
                      labelnextpage="Go to next page"
                      labelpage="[Function]"
                      labelprevpage="Go to previous page"
                      lastnumber="true"
                      lasttext="Â»"
                      limit="5"
                      nexttext="âº"
                      perpage="10"
                      prevtext="â¹"
                      totalrows="0"
                      value="1"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
